src/decays/specializations/Ah/decay_Ah_to_ubaru.inc

```diff
@@ -143,5 +143,21 @@ double CLASSNAME::get_partial_width<PseudoscalarHiggs, bar<UpTypeQuark>::type, U
       result = (1-4.*xOS)*result_DR + 4*xOS*result_OS;
    }
 
+   if (static_cast<bool>(flexibledecay_settings.get(FlexibleDecay_settings::call_higgstools)) ||
```

Comment:
See my comment on `static_cast<bool>`

Reply:
Done in c587517522b7b65208aa28eb3ac3b70a03a3ce92.